features/events/syslog/src/test/resources/etc/syslogd-configuration.xml

```diff
@@ -4,7 +4,7 @@
             syslog-port="10514"
             listen-address="127.0.0.1"
             new-suspect-on-message="false"
-            forwarding-regexp="^.*\s(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"
+            forwarding-regexp="^.*\\s*(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"
```

Comment:
It's kinda correct in both versions. '\s' is a illegal escape char and therefor it's replaced with the string "\\s". To make it safe, one should use the double back-slash.

-- 


-----Original Message-----
From: Seth Leger <notifications@github.com>
To: OpenNMS/opennms <opennms@noreply.github.com>
Sent: Do., 03 Dez. 2015 17:36
Subject: Re: [opennms] Feature refactor syslogd (#484)

> @@ -4,7 +4,7 @@
>              syslog-port="10514"
>              listen-address="127.0.0.1"
>              new-suspect-on-message="false"
> -            forwarding-regexp="^.*\s(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"
> +            forwarding-regexp="^.*\\s*(19|20)\d\d([-/.])(0[1-9]|1[012])\2(0[1-9]|[12][0-9]|3[01])(\s+)(\S+)(\s)(\S.+)"

Is the extra \ here intentional? I think you just meant to add the asterisk to the \s clause.

---
Reply to this email directly or view it on GitHub:
https://github.com/OpenNMS/opennms/pull/484/files#r46575449